Execution of ng add fails on the project using angular cli v11.x

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...

<!-- Check one of the following options with "x" -->
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
- [ ] Performance issue
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Support request
- [ ] Other... Please describe:

## Current behavior
Execution of `ng add` fails on the project using angular cli v11.x.
The installation is successful, but the migration fails.

<!-- Describe how the issue manifests. -->

## Expected behavior
I want the migration is successful.

<!-- Describe what the desired behavior would be. -->

## Minimal reproduction of the problem with instructions
```sh
$ yarn ng add @ngneat/inspector
yarn run v1.22.10
$ ng add @ngneat/inspector
ℹ Using package manager: yarn
✖ Unable to find compatible package.  Using 'latest'.
✖ Package has unmet peer dependencies. Adding the package may not succeed.
✔ Package successfully installed.
An unhandled exception occurred: Cannot find module '@schematics/angular/utility/config'
Require stack:
- /Users/kasaharu/temp/ng-sample/node_modules/@ngneat/inspector/schematics/ng-add/index.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular-devkit/schematics/tools/index.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular/cli/utilities/json-schema.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular/cli/models/command-runner.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular/cli/lib/cli/index.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular/cli/lib/init.js
- /Users/kasaharu/temp/ng-sample/node_modules/@angular/cli/bin/ng
See "/private/var/folders/97/60cgj9ld76d3_kbq79bzqzj40000gn/T/ng-Mq3dbw/angular-errors.log" for further details.
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## What is the motivation / use case for changing the behavior?

<!-- Describe the motivation or the concrete use case. -->

## Environment

### Angular version: X.Y.Z
<!-- Check whether this is still an issue in the most recent Angular version -->

```sh
yarn ng version
yarn run v1.22.10
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 11.2.4
Node: 14.15.1
OS: darwin x64

Angular: 11.2.5
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1102.4
@angular-devkit/build-angular   0.1102.4
@angular-devkit/core            11.2.4
@angular-devkit/schematics      11.2.4
@angular/cli                    11.2.4
@schematics/angular             11.2.4
@schematics/update              0.1102.4
rxjs                            6.6.6
typescript                      4.1.5

✨  Done in 0.66s.
```


### Browser:

- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
### For Tooling issues:
- Node version: v14.15.1
- OS: macOS Catalina v10.15.7

### Others:
<!-- Anything else relevant?  Operating system version, IDE, package manager, HTTP server, ... -->
Not reproduced in Angular v10.x.

```sh
yarn ng add @ngneat/inspector
yarn run v1.22.10
$ ng add @ngneat/inspector
Installing packages for tooling via yarn.
Installed packages for tooling via yarn.
    🔍 Installing packages...
    ✅ Written import statement for "environment"
    ✅ Written import statement for "InspectorModule"
    ✅ Imported "InspectorModule" as non-production in imports
UPDATE src/app/app.module.ts (566 bytes)
✔ Packages installed successfully.
✨  Done in 14.16s.

$ yarn ng version
yarn run v1.22.10
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 10.2.3
Node: 14.15.1
OS: darwin x64

Angular: 10.2.4
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1002.3
@angular-devkit/build-angular   0.1002.3
@angular-devkit/core            10.2.3
@angular-devkit/schematics      10.2.3
@angular/cli                    10.2.3
@schematics/angular             10.2.3
@schematics/update              0.1002.3
rxjs                            6.6.6
typescript                      4.0.7

✨  Done in 0.71s.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.1.2](https://github.com/ngneat/inspector/compare/v1.1.0...v1.1.2) (2021-03-14)
+
 ## [1.1.0](https://github.com/ngneat/inspector/compare/v1.0.1...v1.1.0) (2020-11-06)
 
 

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
     "@schematics/angular": "^10.1.7",
     "ace-builds": "^1.4.12",
     "bootstrap": "^4.5.2",
+    "jsonc-parser": "^3.0.0",
     "rxjs": "~6.6.0",
     "tinykeys": "^1.1.0",
     "tslib": "^2.0.0",

--- a/projects/ngneat/inspector/package.json
+++ b/projects/ngneat/inspector/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@ngneat/inspector",
-  "version": "1.1.0",
+  "version": "1.1.2",
   "schematics": "./schematics/collection.json",
   "scripts": {
     "build": "tsc -p tsconfig.schematics.json",

--- a/projects/ngneat/inspector/schematics/ng-add/index.ts
+++ b/projects/ngneat/inspector/schematics/ng-add/index.ts
@@ -2,11 +2,10 @@ import { Rule, Tree, SchematicsException, chain, SchematicContext } from '@angul
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
 import { insertImport, isImported } from '@schematics/angular/utility/ast-utils';
 import { InsertChange } from '@schematics/angular/utility/change';
-import { getWorkspace } from '@schematics/angular/utility/config';
 import { getAppModulePath } from '@schematics/angular/utility/ng-ast-utils';
 
 import { Schema } from './schema';
-import { addModuleImportToRootModule, getProjectFromWorkspace, getSourceFile } from './utils';
+import { addModuleImportToRootModule, getProjectFromWorkspace, getSourceFile, getWorkspace } from './utils';
 import { hasNgModuleImport } from './utils/ng-module-imports';
 import { targetBuildNotFoundError } from './utils/project-targets';
 const importModuleSet = [

--- a/projects/ngneat/inspector/schematics/ng-add/utils/index.ts
+++ b/projects/ngneat/inspector/schematics/ng-add/utils/index.ts
@@ -1,11 +1,3 @@
-/**
- * @license
- * Copyright Google LLC All Rights Reserved.
- *
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://angular.io/license
- */
-
 import * as ts from 'typescript';
 import { addImportToModule } from '@schematics/angular/utility/ast-utils';
 import { Change, InsertChange } from '@schematics/angular/utility/change';
@@ -16,6 +8,7 @@ import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
 import { Rule, SchematicContext, SchematicsException, Tree } from '@angular-devkit/schematics';
 import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
 import { WorkspaceProject, WorkspaceSchema } from '@schematics/angular/utility/workspace-models';
+import { parse } from 'jsonc-parser';
 
 export function installPackageJsonDependencies(): Rule {
   return (host: Tree, context: SchematicContext) => {
@@ -173,4 +166,20 @@ export function getSourceFile(host: Tree, path: string) {
   const content = buffer.toString();
 
   return ts.createSourceFile(path, content, ts.ScriptTarget.Latest, true);
+}
+
+export function getWorkspacePath(host: Tree) {
+  const possibleFiles = ['/angular.json', '/.angular.json'];
+  const path = possibleFiles.filter((filePath) => host.exists(filePath))[0];
+  return path;
+}
+
+export function getWorkspace(host: Tree) {
+  const path = getWorkspacePath(host);
+  const configBuffer = host.read(path);
+  if (configBuffer === null) {
+    throw new SchematicsException(`Could not find (${path})`);
+  }
+  const content = configBuffer.toString();
+  return parse(content);
 }

--- a/projects/ngneat/inspector/src/public-api.ts
+++ b/projects/ngneat/inspector/src/public-api.ts
@@ -14,7 +14,7 @@
 // export * from './lib/components/inspector-panel/tabs/properties/properties.component';
 // export * from './lib/components/inspector-panel/inspector-panel.component';
 // export * from './lib/directives/field-host.directive';
-// export * from './lib/inspector.component';
+export * from './lib/inspector.component';
 export * from './lib/inspector.module';
 // export * from './lib/inspector.model';
 // export * from './shared/helpers';

